### PR TITLE
[tests-only][full-ci] added issue tag `@issue-7257` on added scenarios

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
In this PR, added issue tag `@issue-7257` on the added scenario in PR: https://github.com/owncloud/ocis/pull/8088

## Related Issue

- https://github.com/owncloud/ocis/issues/8044

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 1]
```diff
@@ -434,3 +434,81 @@ Feature: upload file
     Then the HTTP status code should be "204"
     And for user "Brian" the content of the file "/textfile.txt" of the space "new-space" should be ""
     And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
+
+
+  Scenario Outline: user replaces a file with an empty file
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:439]: Can we demonstrate this bug report in scenario?
https://github.com/owncloud/ocis/issues/8003#issuecomment-1867226950

Comment (PrajwolAmatya) [reply tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:439]: No, through API replacing a file with an empty file works fine. The bug only seems to exist on the web.

Comment (saw-jan) [reply tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:439]: it must be a TUS upload

Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:439]: Could you add issue tags here and in previous scenarios?
```suggestion
   @issue-7257
  Scenario Outline: user replaces a file with an empty file
```

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 2]
```diff
@@ -434,3 +434,81 @@ Feature: upload file
     Then the HTTP status code should be "204"
     And for user "Brian" the content of the file "/textfile.txt" of the space "new-space" should be ""
     And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
+
+  @issue-7275
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:438]:
```suggestion
  @issue-7257
```

**Review by saw-jan — APPROVED** [review]

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 3]
```diff
@@ -434,3 +434,81 @@ Feature: upload file
...
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "file with content" to "/textfile.txt"
+    And user "Alice" has shared a space "new-space" with settings:
+      | shareWith | Brian  |
+      | role      | editor |
+    When user "Brian" uploads a file "filesForUpload/zerobyte.txt" to "/textfile.txt" in space "new-space" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Brian" the content of the file "/textfile.txt" of the space "new-space" should be ""
+    And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:514]: These scenarios behave the same as the above scenarios added in https://github.com/owncloud/ocis/pull/8088. So I think we should not add these as duplication.

@PrajwolAmatya Could you remove these new scenarios and add only tags change in this PR?

**Review by saw-jan — CHANGES_REQUESTED** [review]

**Review by saw-jan — APPROVED** [review]

**Review by grgprarup — APPROVED** [review]

**Pull request merged**